src/Movie.ts

```diff
@@ -0,0 +1,7 @@
+export interface Movie {
+  title: string;
+  description: string;
```

Comment:
description?: string;
All the fields in the movie except the description are required.